crates/bevy_reflect/src/serde/de.rs

```diff
@@ -292,6 +293,9 @@ impl<'a, 'de> Visitor<'de> for UntypedReflectDeserializerVisitor<'a> {
             registration,
             registry: self.registry,
         })?;
+        if map.next_key::<IgnoredAny>()?.is_some() {
+            return Err(Error::invalid_length(2, &"a single entry"));
+        }
```

Comment:
Super minor nit: Could we add some spaces around this block to help set it apart? 😅 
```suggestion

        if map.next_key::<IgnoredAny>()?.is_some() {
            return Err(Error::invalid_length(2, &"a single entry"));
        }

```